.gitignore

```diff
@@ -6,9 +6,13 @@ tmp
 .DS_Store
 .idea
 public/build/
-npm-debug.log
+npm-debug.log*
 config/*.local.*
 public/*
 !public/index.html
```

Comment:
I think this is ok, we only should exclude index.html (no other html files should be there).